.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,221 @@
+name: Docker
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  push:
+    # TODO: Change back to main
+    branches: [ "docker" ]
```

Comment:
TODO: This has to be addressed before merge.

Reply:
Awesome. Thank you so much for putting in the work. I'll merge this.